web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -8,142 +8,51 @@
       @close="edit = false"
     />
     <template v-else>
-      <v-container>
+      <v-toolbar class="grey darken-2 white--text">
+        <v-btn
+          icon
+          @click="navigateBack"
+        >
+          <v-icon color="white">
+            mdi-arrow-left
+          </v-icon>
+        </v-btn>
+        <v-toolbar-title>
+          Dandiset Dashboard
+        </v-toolbar-title>
+        <v-spacer />
+        <DandisetSearchField />
+        <v-btn
+          icon
+          @click="detailsPanel = !detailsPanel"
+        >
+          <v-icon color="white">
+            <template v-if="detailsPanel">
+              mdi-chevron-right
+            </template>
+            <template v-else>
+              mdi-chevron-left
+            </template>
+          </v-icon>
```

Comment:
This `>` is situated right next to the search box and looks exactly like a "submit the search query" button. The figma makes it a little clearer that it's an expand/collapse button, but not a whole lot.
I don't see an obvious way to change it, but the way it is now is very counterintuitive.

Reply:
You also cleverly hide the details view completely at lower widths, but the chevron is still visible. That should also be hidden if the thing it expands/collapses is never shown.

Reply:
It's actually not hidden, it's just pushed below. I'm not sure what behavior we want here exactly, i.e. where to show this data or if to show it at all at lower widths

Reply:
I think making it fill the width and putting it at the top would be reasonably intuitive. The open/close button is still confusing though.

Reply:
> I think making it fill the width and putting it at the top would be reasonably intuitive.

This has been addressed in 4bd2df5, I encourage you to try it out. I'm not sure what to do about the open/close button, but I think it should be left as a follow-up issue, so we can get this merged.

Reply:
How about up/down chevrons instead of left/right? After toying with it they seem to convey the same thing without being as suggestive of a "Search" button.
I do think the button needs some more work, though. This is just a bandaid.

Reply:
cc015b7